pkg/process/events/model/model_common.go

```diff
@@ -3,6 +3,8 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
+//go:generate go run github.com/tinylib/msgp -tests=false
+
```

Comment:
Don't we need to add the `"msg:"` tags to the `ProcessEvent` struct here as it has been done for `ProcessMonitoringEvent `?

Reply:
We don't since all the fields are exported (the msg name will just be the name of the fields). But we could add them for clarity if you want

Reply:
Oh, I see. Thanks for clarifying it. I think that it's worth adding the tags to make it more explicit that we're using mspack to serialize them.

Reply:
I added them in the last commit